recipe-finder-fe/src/Components/Header/index.tsx

```diff
@@ -0,0 +1,17 @@
+interface HeaderProp {
```

Comment:
Props are always called Props, even if there's one in the object - there may be more later

Reply:
Done